### PR TITLE
feat: Introduce the new AssetFiltersModal

### LINKED ISSUE #1219: [2.1] Mobile improvements:  Market filter is visible by default
## 🎉 Description

* [X] Market filter is visible by default as the title describes. (done in a previos PR)
* [ ] Add the new filters icon button that will open the new filters modal
* [ ] Style the new store/listing toggle button to look like the design from below 

* [ ] 🐛 This is a bug report.
* [X] 📈 This is a feature request.

### PR DESCRIPTION
**Pull request by juanmahidalgo**
Closes #1219

### CODE REVIEW

File: webapp/src/components/Vendor/decentraland/NFTTopbar/NFTTopbar.tsx [hunk 1]
```diff
@@ -135,12 +156,27 @@ export const NFTTopbar = ({
...
-          />
+          <div className={styles.rightOptionsContainer}>
+            <Dropdown
+              direction="left"
+              value={sortByValue}
+              options={orderByDropdownOptions}
+              onChange={handleOrderByDropdownChange}
+            />
+            {isMobile ? (
+              <div
```
Comment (meelrossi) [inline webapp/src/components/Vendor/decentraland/NFTTopbar/NFTTopbar.tsx:167]: can we move this to a button and the div inside to an <i> 😬

File: webapp/src/components/Vendor/decentraland/NFTTopbar/NFTTopbar.module.css [hunk 2]
```diff
@@ -77,3 +77,29 @@
...
+  background-repeat: no-repeat;
+  background-position: center;
+  display: inline-block;
+  margin-left: 24px;
+}
+
+.openFilters.active {
+  background-image: url(../../../../images/filters-active.svg);
+  background-size: 20px;
+}
```
Comment (meelrossi) [inline webapp/src/components/Vendor/decentraland/NFTTopbar/NFTTopbar.module.css:105]: new line here?

File: webapp/src/components/Vendor/decentraland/NFTTopbar/NFTTopbar.tsx (hunk 1 above)
Comment (juanmahidalgo) [reply webapp/src/components/Vendor/decentraland/NFTTopbar/NFTTopbar.tsx]: nice idea! updated!

File: webapp/src/components/Vendor/decentraland/NFTTopbar/NFTTopbar.module.css (hunk 2 above)
Comment (juanmahidalgo) [reply webapp/src/components/Vendor/decentraland/NFTTopbar/NFTTopbar.module.css]: absolutely! added

**Review by meelrossi — APPROVED** [review]

**Pull request merged**